package.json

```diff
@@ -11,6 +11,12 @@
     "electron-lint-markdown-standard": "./dist/bin/lint-markdown-standard.js",
     "electron-lint-markdown-ts-check": "./dist/bin/lint-markdown-ts-check.js"
   },
+  "files": [
+    "configs",
+    "dist",
+    "lib",
```

Comment:
Question: does `lib` actually need to be published?

Reply:
Good eye. `lib` is published because I have some vague intentions of using the code in there as building blocks for more specific linting scripts on the `electron/electron` repo without tying this package too specifically to that repo. I have plans to make this package more generic so that the Markdown code block type checking feature could be used in other projects, but that'll require some breaking changes so I'm gonna group together a bunch of breaking changes in the future to do them all at once.

Also, similarly to Forge, this package ships `.js.map` files in `dist/` which reference `lib/`. That may be revisited in the future, but it's a standard `tsconfig.json` option across a lot of our Ecosystem WG repos at the moment.